### PR TITLE
test(exerciese): Remove deprecated cssSelector option

### PR DESCRIPTION
**Pull request by Droogans**
JIRA: https://jira.rax.io/browse/FRMW-211

### LGTMs
- [ ] Dev LGTM
- [ ] QE/Dev LGTM

### CODE REVIEW

File: src/components/rxCollapse/scripts/rxCollapse.exercise.js [hunk 1]
```diff
@@ -37,11 +36,6 @@ exports.rxCollapse = function (options) {
             } else {
                 component = rxCollapse.initialize();
```
Comment (CITguy) [inline src/components/rxCollapse/scripts/rxCollapse.exercise.js]: I'd default `options.instance` to `rxCollapse.initialize()` for consistency.

File: src/components/rxCollapse/scripts/rxCollapse.exercise.js [hunk 2]
```diff
@@ -8,7 +8,6 @@ var rxCollapse = require('./rxCollapse.page').rxCollapse;
  * @exports exercise/rxCollapse
  * @param {Object} [options] - Test options. Used to build valid tests.
  * @param {rxCollapse} [options.instance=rxCollapse.initialize()] - Component to exercise.
```
Comment (CITguy) [inline src/components/rxCollapse/scripts/rxCollapse.exercise.js]: I'd link to the `initialize` docs for consistency.

File: src/components/rxOptionTable/scripts/rxOptionTable.exercise.js [hunk 3]
```diff
@@ -6,7 +6,6 @@ var rxOptionTable = require('./rxOptionTable.page').rxOptionTable;
  * @exports exercise/rxOptionTable
  * @param {Object} [options] - Test options. Used to build valid tests.
  * @param {rxOptionTable} [options.instance=rxOptionTable.initialize] - Component to exercise.
```
Comment (CITguy) [inline src/components/rxOptionTable/scripts/rxOptionTable.exercise.js]: I'd link to the `initialize` docs for consistency.

File: src/components/rxPaginate/scripts/rxPaginate.exercise.js [hunk 4]
```diff
@@ -7,7 +7,6 @@ var rxPaginate = require('./rxPaginate.page').rxPaginate;
  * @exports exercise/rxPaginate
  * @param {Object} [options] - Test options. Used to build valid tests.
  * @param {rxPaginate} [options.instance=rxPaginate.initialize] - Component to exercise.
```
Comment (CITguy) [inline src/components/rxPaginate/scripts/rxPaginate.exercise.js]: Please link to `initialize` docs for consistency.

File: src/components/rxSearchBox/scripts/rxSearchBox.exercise.js [hunk 5]
```diff
@@ -7,7 +7,6 @@ var rxSearchBox = require('./rxSearchBox.page').rxSearchBox;
  * @exports exercise/rxSearchBox
  * @param {Object} [options] - Test options. Used to build valid tests.
  * @param {rxSearchBox} [options.instance=rxSearchbox.initialize()] - Component to exercise.
```
Comment (CITguy) [inline src/components/rxSearchBox/scripts/rxSearchBox.exercise.js]: Please link to `initialize` docs for consistency.

File: src/components/rxToggleSwitch/scripts/rxToggleSwitch.exercise.js [hunk 6]
```diff
@@ -7,7 +7,6 @@ var rxToggleSwitch = require('./rxToggleSwitch.page').rxToggleSwitch;
  * @exports exercise/rxToggleSwitch
  * @param {Object} [options] - Test options. Used to build valid tests.
  * @param {rxToggleSwitch} [options.instance=rxToggleSwitch.initialize()] - Component to exercise.
```
Comment (CITguy) [inline src/components/rxToggleSwitch/scripts/rxToggleSwitch.exercise.js]: Please link to `initialize` docs.

**Comment by Droogans** [comment]
I didn't mean to push these commit up from `Droogans:2.x`.

**Comment by Droogans** [comment]
Hate to do this to you, but I'm going to close this and re-open it on another branch other than 2.x!

I apologize.

**Pull request closed without merging**